Patches/Archotech Expanded/AE_Items_Resource_Stuff.xml

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+<Operation Class="PatchOperationFindMod">
+	<mods>
+		<li>Archotech Expanded</li>
+	</mods>
+	<match Class="PatchOperationSequence">
+		<operations>
+	<!-- Atto -->
+
+	<li Class="PatchOperationAdd">
+		<xpath>Defs/ThingDef[defName="Attomachinery"]/statBases</xpath>
+		<value>
+			<Bulk>0.03</Bulk>
+		</value>
+	</li>
+
+	<li Class="PatchOperationAdd">
+		<xpath>Defs/ThingDef[defName="Attomachinery"]/stuffProps/categories</xpath>
+		<value>
+			<li>Metallic_Weapon</li>
+			<li>Steeled</li>
+		</value>
+	</li>
+	
+	<li Class="PatchOperationAdd">
+		<xpath>Defs/ThingDef[defName="Attomachinery"]/stuffProps/statFactors</xpath>
+		<value>
+			<MeleePenetrationFactor>50</MeleePenetrationFactor>
+		</value>
+	</li>
+	
+	<li Class="PatchOperationReplace">
+		<xpath>Defs/ThingDef[defName="Attomachinery"]/statBases/StuffPower_Armor_Sharp</xpath>
+		<value>
+			<StuffPower_Armor_Sharp>50</StuffPower_Armor_Sharp>
+		</value>
+	</li>
+
+	<li Class="PatchOperationReplace">
+		<xpath>Defs/ThingDef[defName="Attomachinery"]/statBases/StuffPower_Armor_Blunt</xpath>
+		<value>
+			<StuffPower_Armor_Blunt>50</StuffPower_Armor_Blunt>
+		</value>
+	</li>
+	
+	<!-- Pico -->
+	
+	<li Class="PatchOperationAdd">
+		<xpath>Defs/ThingDef[defName="Picomachinery"]/statBases</xpath>
+		<value>
+			<Bulk>0.03</Bulk>
+		</value>
+	</li>
+
+	<li Class="PatchOperationAdd">
+		<xpath>Defs/ThingDef[defName="Picomachinery"]/stuffProps/categories</xpath>
+		<value>
+			<li>Metallic_Weapon</li>
+			<li>Steeled</li>
+		</value>
+	</li>
+	
+	<li Class="PatchOperationAdd">
+		<xpath>Defs/ThingDef[defName="Picomachinery"]/stuffProps/statFactors</xpath>
+		<value>
+			<MeleePenetrationFactor>25</MeleePenetrationFactor>
+		</value>
+	</li>
+	
+	<li Class="PatchOperationReplace">
+		<xpath>Defs/ThingDef[defName="Picomachinery"]/statBases/StuffPower_Armor_Sharp</xpath>
+		<value>
+			<StuffPower_Armor_Sharp>25</StuffPower_Armor_Sharp>

```

Comment:
I just want to confirm this is intended to be this high, because it's kind of insane. Making a flak vest out of this stuff will give you **200**mm of sharp armor. Just for comparison; a German King Tiger tank had about 150mm.

I understand it's probably intended to be overpowered, but I do want you to consider what you think _should_ be able to beat it.

Reply:
Yes, it's supposed to be this way